src/assets/custom/css/_sass/_typography.scss

```diff
@@ -0,0 +1,39 @@
+@mixin bold {
+  font-family: "Proxima Nova";
+  font-weight: 700;
+}
+
+@mixin reg {
+  font-family: "Proxima Nova";
+  font-weight: 400;
+}
+
+@mixin freed-bold {
+  @include bold;
+  font-size: 28px;
+  line-height: 38px;
+}
+
+@mixin freed-bold__responsive {
+  @include freed-bold;
+
+  @include medium-large-and-extra-large {
+    font-size: 44px;
+    line-height: 1.25;
+  }
+}
+
+@mixin base__reg {
+  @include bold;
+  margin-bottom: 0;
+  margin-top: 0;
+  line-height: 1.5;
+
+  @include small {
+    font-size: 18px;
+  }
+
+  @include medium-large-and-extra-large {
+    font-size: 28px;
+  }
+}
```

Comment:
Make 'freed-bold' and 'base__reg' consistent.

Reply:
I ended up deleting/stashing the `base-reg` styles because they're not being used anywhere just yet. I will add them in and make them consistent when I first use them (on a future branch)